plugins/module_utils/podman/podman_container_lib.py

```diff
@@ -1529,10 +1529,14 @@ def ensure_image_exists(module, image, module_params):
         return image_actions
     if not image:
         return image_actions
-    rc, out, err = module.run_command([module_exec, 'image', 'exists', image])
+    image_exists_cmd = [module_exec, 'image', 'exists', image]
+    rc, out, err = module.run_command(image_exists_cmd)
```

Comment:
What is the purpose of this change?

Reply:
mostly to make it coherent with the other modified command, nothing specific!